.buildkite/pipeline.verify.yml

```diff
@@ -236,6 +236,12 @@ steps:
   - label: ":hammer: E2E Tests"
     command:
       - make test-e2e
+      # For most realistic results in this assertion, run it after an E2E build.
+      # This will:
+      # - create some artifacts in dist
+      # - create some engagement-view npm/yarn state
+      # etc.
+      - .buildkite/scripts/assert_docker_context_size.sh
```

Comment:
I think we're going to eventually want to break this out into a separate step that a) builds the images (or at least builds all the _prerequisites_ that we will ultimately stuff into the images) and then b) runs the assertion script. This would allow us to soft-fail the assertion, but continue to hard-fail the e2e tests.

As it is, we're coupling two things that probably shouldn't be coupled; it'd suck to have everything pass, only to fail because you forgot to ignore a new directory. We'd certainly like to be made aware of that, but having it be a hard failure strikes me as a bit harsh.

Reply:
I agree, but I wasn't sure that it's worth the extra full build _just_ for that :|